packages/core/addon/components/navi-visualization-config/series-chart.ts

```diff
@@ -177,10 +194,8 @@ class NaviVisualizationConfigSeriesChartComponent extends Component {
    */
   @action
   onUpdateChartMetric(metric) {
-    const newConfig = copy(this.seriesConfig);
-    set(newConfig, `metric`, metric);
-    this.onUpdateConfig(newConfig);
+    const newConfig = copy(this.args.seriesConfig);
+    newConfig.metric.metric = metric.cid;
```

Comment:
Per Jon's earlier comment
```suggestion
    newConfig.metricCid= metric.cid;
```